### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by JessicaDosseh**
# README.MD PREVIEW

<div align="center">
  
  
  <hr height="0.5px" />
  
  <br/>
  
  <h1> Cosmos Hub (Gaia) </h1>
  <p> The Cosmos Hub is the first of thousands of interconnected blockchains that will eventually comprise the Cosmos Network. </p>
  
  <a href="https://twitter.com/cosmoshub">Tweet</a>
  <br/>
</div>

<hr/>

<div align="center">
  
</div>

<br/>

### 🤔 — Why should you be interested in the Cosmos Hub 
___

The Cosmos Hub is built using the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk) and to build a binary called `gaiad` (Gaia Daemon). The Cosmos Hub and other fully sovereign Cosmos SDK blockchains interact with one another using a protocol called [IBC](https://github.com/cosmos/ibc) that enables Inter-Blockchain Communication. In order to understand what the Cosmos Hub is you can read this [introductory explanation](https://hub.cosmos.network/main/hub-overview/overview.html).

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
| [codecov](https://codecov.io/gh/cosmos/gaia) | [Go Report Card](https://goreportcard.com/report/github.com/cosmos/gaia) | [license](https://github.com/cosmos/gaia/blob/main/LICENSE) | [LoC](https://github.com/cosmos/gaia) | [GolangCI](https://golangci.com/r/github.com/cosmos/gaia) |

<br/>
<br/>

### ⚡ — Documentation & Introduction
___

You can find an introduction to the Cosmos Hub and how to use gaia as a delegator, validator or node operator as well as how governance on the Cosmos Hub works in the [documentation](https://hub.cosmos.network/main/hub-overview/overview.html). 

<br/>
<br/>

### 👤 — Node Operators
___
If you're interested in running a node on the current Cosmos Hub, check out the docs to [Join the Cosmos Hub Mainnet](https://github.com/cosmos/gaia/blob/main/docs/gaia-tutorials/join-mainnet.md).

<br/>
<br/>

### 🗣️ — Validators
___

If you're running a validator node on the Cosmos Hub, reach out to a Janitor on the [Cosmos Developers Discord](https://discord.gg/cosmosnetwork) to join the `#validators-verified` channel.

<br/>
<br/>

### 👥 — Testnet
___

To participate in or utilize the current Cosmos Hub testnet that upgrades `cosmoshub-4` from gaia v5.0.x to gaia v6.0.x, take a look at the [vega-testnet](https://github.com/cosmos/vega-test) repository. This testnet is for the Vega Upgrade expected in Q4 2021. For future upgrades of the Cosmos Hub take a look at the [roadmap](https://github.com/cosmos/gaia/blob/main/docs/roadmap/cosmos-hub-roadmap-2.0.md).

<br/>
<br/>

### 🌐 — Roadmap
___

For an overview of upcoming changes to the Cosmos Hub take a look at the [Roadmap](https://github.com/cosmos/gaia/blob/main/docs/roadmap/cosmos-hub-roadmap-2.0.md).

<br/>
<br/>

### 🗄️ — Archives & Genesis
___

With each version of the Cosmos Hub, the chain is restarted from a new Genesis state. 
Mainnet is currently running as `cosmoshub-4`. Archives of the state of `cosmoshub-1`, `cosmoshub-2`, and `cosmoshub-3` are available [here](./docs/resources/archives.md).

If you are looking for historical genesis files and other data [`cosmos/mainnet`](http://github.com/cosmos/mainnet) is an excellent resource. Snapshots are also available at [cosmos.quicksync.io](https://cosmos.quicksync.io).

<br/>
<br/>

### 🤝 — How to contribute 
___

Check out [contributing.md](CONTRIBUTING.md) for our guidelines & policies for how we develop the Cosmos Hub. Thank you to all those who have contributed!

<br/>
<br/>

### 💬 — Talk to us
___

We have active, helpful communities on Twitter, Discord, and Telegram.

|    |    |
| -- | -- |
| Cosmos Developers Discord | <a href="https://discord.gg/cosmosnetwork">Discord</a> |
| Cosmos Twitter | <a href="https://twitter.com/cosmos">Tweet</a> |
| Cosmos Telegram | <a href="https://t.me/cosmosproject">Telegram</a> |

For updates on the Cosmos Hub team's activities follow us on the [Cosmos Hub Twitter](https://twitter.com/cosmoshub) account.

<br/>
<br/>

### 👏 — Supporters
___

[Stargazers repo roster for @cosmos/gaia](https://github.com/cosmos/gaia/stargazers)
[Forkers repo roster for @cosmos/gaia](https://github.com/cosmos/gaia/network/members)

<br/>

<p align="center"><a href="https://github.com/nastyox/Rando.js#nastyox">Animated footer bars</a></p>

<br/>

<p align="center"><a href="https://github.com/cosmos/gaia#">Back to top</a></p>

### CODE REVIEW

**Review by zmanian — APPROVED** [review]

**Comment by zmanian** [comment]
This looks great.

File: README.md [hunk 1]
```diff
@@ -1,50 +1,140 @@
...
-![image](https://miro.medium.com/max/2000/1*DHtmSfS_Efvuq8n2LAnhkA.png)
+<div align="center">
+  <img src="https://www.notion.so/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2F643ab9b2-4062-4367-99f6-59c0f7507e22%2FGroup_1041.png?table=block&id=23fcc36a-07b6-46a5-84d6-2f4a5569cd6e&spaceId=2fa227ac-dee8-4655-ad64-fe3c359def5a&width=250&userId=b57d8bef-0356-4537-8ae8-7251a098e5b3&cache=v2" width="100" height="100" />
+  
+  <hr height="0.5px" />
+  
+  <br/>
+  
+  <h1> Cosmos Hub (Gaia) </h1>
+  <p> The Cosmos Hub is the first of thousands of interconnected blockchains that will eventually comprise the Cosmos Network. </p>
```
Comment (okwme) [inline README.md]:
```suggestion
  <p> The Cosmos Hub is the first of an exploding number of interconnected blockchains that comprise the Cosmos Network. </p>
```

File: README.md [hunk 2]
```diff
@@ -1,50 +1,140 @@
...
+  <img src="https://miro.medium.com/max/2000/1*DHtmSfS_Efvuq8n2LAnhkA.png" />
+</div>
+
+<br/>
+
+
+### 🤔 — Why should you be interested in the Cosmos Hub 
+___
 
 The Cosmos Hub is built using the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk) and to build a binary called `gaiad` (Gaia Daemon). The Cosmos Hub and other fully sovereign Cosmos SDK blockchains interact with one another using a protocol called [IBC](https://github.com/cosmos/ibc) that enables Inter-Blockchain Communication. In order to understand what the Cosmos Hub is you can read this [introductory explanation](https://hub.cosmos.network/main/hub-overview/overview.html).
```
Comment (okwme) [inline README.md]:
```suggestion
The Cosmos Hub is built using the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk) and compiled to a binary called `gaiad` (Gaia Daemon). The Cosmos Hub and other fully sovereign Cosmos SDK blockchains interact with one another using a protocol called [IBC](https://github.com/cosmos/ibc) that enables Inter-Blockchain Communication. In order to understand what the Cosmos Hub is you can read this [introductory explanation](https://hub.cosmos.network/main/hub-overview/overview.html).
```

File: README.md [hunk 3]
```diff
@@ -1,50 +1,140 @@
...
 
-## Documentation & Introduction
+<br/>
+<br/>
+
+
+### ⚡ — Documentation & Introduction
+___
+
 You can find an introduction to the Cosmos Hub and how to use gaia as a delegator, validator or node operator as well as how governance on the Cosmos Hub works in the [documentation](https://hub.cosmos.network/main/hub-overview/overview.html).
```
Comment (okwme) [inline README.md]:
```suggestion
You can find an introduction to the Cosmos Hub and how to use the `gaiad` binary as a delegator, validator or node operator as well as how governance on the Cosmos Hub works in the [documentation](https://hub.cosmos.network/main/hub-overview/overview.html). 
```

**Review by okwme — APPROVED** [review]
great work @JessicaDosseh !

**Comment by okwme** [comment]
great work @JessicaDosseh !

**Pull request merged**